README.md

```diff
@@ -274,6 +274,29 @@ Get the digest for your container _without_ pulling it using the [crane](https:/
 IMAGE="${IMAGE}@"$(crane digest "${IMAGE}")
 ```
 
+**Optional:** 
```

Comment:
Let's remove this since it's going to be removed int he next PR when we introduce the `--provenance-registry`